test/unit/hooks-worker-client-test.coffee

```diff
@@ -124,10 +134,10 @@ describe 'Hooks worker client', ->
         setTimeout ->
           hooksWorkerClient.stop (stopError) ->
             return done stopError if stopError
-            assert.isDefined runner.hookHandlerError
+            assert.ok runner.hookHandlerError
```

Comment:
Interesting, https://github.com/chaijs/chai/commit/e5071edd8b3488d1d074c1bb9a6f6b36a64a4ff0 happened 2 years ago, but I didn't notice. I think I use `assert.ok` or `assert.isOk` unknowingly interchangeably for the whole time.

In that case, it would be probably worth to unify this across the Dredd codebase, but I wouldn't do that in this PR. Thanks for pointing this out!